R/wage_inflator.R

```diff
@@ -2,8 +2,8 @@
 #' @description Predicts the inflation of hourly rates of pay, between two financial years.
 #' 
 #' @param wage The amount to be inflated (1 by default).
-#' @param from_fy A character vector of the form "2012-13" representing the FY ending that the wage index is to be taken (i.e. Q4 in that year). FY year must be 1996-97 or later.
-#' @param to_fy The FY ending that the wage index is to be taken.
+#' @param \code{from_fy} A character vector of the form "2012-13" representing the FY ending that the wage index is to be taken (i.e. Q4 in that year). FY year must be 1996-97 or later.If both from_fy and to_fy are not given, the previous financial year is the default value for from_fy.  
+#' @param \code{to_fy} The FY ending that the wage index is to be taken. If both from_fy and to_fy are not given, the current financial year is the default value for to_fy.
```

Comment:
No `\code` for the first word after `@param`.

Reply:
Don't exactly get what you mean. Should I remove the \code{ } around from_fy?

Reply:
Yes instead of

```r
#' @param \code{from_fy} A character vector of the form "2012-13" representing the FY ending that the wage index is to be taken (i.e. Q4 in that year). FY year must be 1996-97 or later.If both from_fy and to_fy are not given, the previous financial year is the default value for from_fy.  
#' @param \code{to_fy} The FY ending that the wage index is to be taken. If both from_fy and to_fy are not given, the current financial year is the default value for to_fy.
```

do

```r
#' @param from_fy A character vector of the form "2012-13" representing the FY ending that the wage index is to be taken (i.e. Q4 in that year). FY year must be 1996-97 or later. If both from_fy and to_fy are not given, the previous financial year is the default value for \code{from_fy}.  
#' @param to_fy The FY ending that the wage index is to be taken. If both \code{from_fy} and \code{to_fy} are not given, the current financial year is the default value for \code{to_fy}.

```

Reply:
Done in commit (ac39e93)[https://github.com/HughParsonage/grattan/pull/97/commits/ac39e931129700474f76472d4a0348b8cf004cab]